packages/grafana-eslint-rules/tests/no-border-radius-literal.test.js

```diff
@@ -32,27 +36,27 @@ ruleTester.run('eslint no-border-radius-literal', noBorderRadiusLiteral, {
   invalid: [
     {
       code: `css({ borderRadius: '2px' })`,
-      errors: [
-        {
-          message: 'Prefer using theme.shape.radius tokens instead of literal values.',
-        },
-      ],
+      errors: [expectedError],
     },
     {
       code: `css({ lineHeight: 1 }, { borderRadius: '2px' })`,
-      errors: [
-        {
-          message: 'Prefer using theme.shape.radius tokens instead of literal values.',
-        },
-      ],
+      errors: [expectedError],
     },
     {
       code: `css([{ lineHeight: 1 }, { borderRadius: '2px' }])`,
-      errors: [
-        {
-          message: 'Prefer using theme.shape.radius tokens instead of literal values.',
-        },
-      ],
+      errors: [expectedError],
+    },
+    {
+      name: 'nested classes',
```

Comment:
FYI @ashharrison90 I pushed a test for the nested classes case that the rule now catches